### PR TITLE
Feature/customer.html

### PR DESCRIPTION
**Pull request by BarbaraIgnaciuk-Charkiewicz**

### CODE REVIEW

File: src/partials/customer.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<section class="customer">
```
Comment (AnnaWitt) [inline src/partials/customer.html:1]: add container

**Review by LukaszCiaston-coder — CHANGES_REQUESTED** [review]
delete files main.css and main.css.map

File: src/partials/customer.html [hunk 2]
```diff
@@ -1,20 +1,14 @@
-<section class="customer">
-    <img class="customer__image"
-    alt="customer photo"
-    src="../images/testimonials-desk-x1.png"
-    srcset="
-    /images/testimonials-desk-x2.png 2x,
-    /images/testimonials-desk-x3.png 3x"
-  />
+<section class="customer container">
+    <img class="customer__image" alt="customer photo" src="../images/testimonials-desk-x1.png" srcset="/images/testimonials-desk-x2.png 2x, /images/testimonials-desk-x3.png 3x"/>
```
Comment (AnnaWitt) [inline src/partials/customer.html:2]: check srcset source line

**Review by LukaszCiaston-coder — CHANGES_REQUESTED** [review]
delete files main.css and main.css.map

**Review by AnnaWitt — APPROVED** [review]

**Comment by BarbaraIgnaciuk-Charkiewicz** [comment]
Usunęłam main.css i main.css.map

**Review by AnnaWitt — APPROVED** [review]

**Pull request closed without merging**